### PR TITLE
updated readme.md

### PR DESCRIPTION
**Pull request by mprakhar1705**
1. changed medium[4] to medium.
2. added spaces after (.) and (,)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,20 +10,20 @@ Content to be unblocked from Cyberoam
...
 
 ### 3.Quora CDN
 - **URL**:https://www.quora.com/
-- **About**: Quora is a question-answer website where people can get answers and opinions on the questions they ask to the community.Its a great platform where the society interacts and solves its problems.A lot of writers on it are masters of their domain including Prof.Thomas Cormen,Steve Wozniak,Jimmy Wales etc.Their answers on various questions are very inspiring and a treat to read and follow.
+- **About**: Quora is a question-answer website where people can get answers and opinions on the questions they ask to the community. Its a great platform where the society interacts and solves its problems.A lot of writers on it are masters of their domain including Prof.Thomas Cormen, Steve Wozniak, Jimmy Wales etc.Their answers on various questions are very inspiring and a treat to read and follow.
```
Comment (ryzokuken) [inline README.md]: 1. `s/Its/It's`
2. A space before "A lot of writers".
3. A space after "Prof."
4. A space before "Their answers".

**Review by ryzokuken — CHANGES_REQUESTED** [review]
Looks great, just a few more changes. Thanks for doing this.

**Comment by ryzokuken** [comment]
@nightwarrior-xxx @sidntrivedi012 I don't exactly see the content you have written before merging, but the application would include all this, so please be more vigilant from now on.

File: README.md (hunk 1 above)
Comment (sidntrivedi) [reply README.md]: @ryzokuken On it.Adding changes.

**Review by sidntrivedi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ryzokuken) [reply README.md]: @sidntrivedi012 You could either add them to this one or @tonystark97 will.

**Comment by mprakhar1705** [comment]
Commited some more changes. Kindly look into it @ryzokuken

**Review by sidntrivedi — APPROVED** [review]

**Review by ryzokuken — APPROVED** [review]
Looks great, thanks!

File: README.md (hunk 1 above)
Comment (sidntrivedi) [reply README.md]: Some changes I think are to be made-
1. Highlighting the port numbers using grave accent.For example: `Port 25` .
2. Adding a whitespace between SMTP and (Simple Mail Transfer Protocol).
3. Adding url to glitch.me and glitch.com.
4. Removal of :- after the IRC heading.
@ryzokuken PTAL.

Comment (ryzokuken) [reply README.md]: @sidntrivedi012 please make a PR for the same, I'll review and merge them.

Comment (sidntrivedi) [reply README.md]: @ryzokuken Ok.

**Pull request merged**